release-team/role-handbooks/ci-signal/README.md

```diff
@@ -119,6 +149,14 @@ This is when things really begin to ramp up in the release cycle with active bug
   - you need not monitor master-blocking on a daily basis. It is, however, worth keeping an eye on master-blocking especially before risky cherry-picks make their way into the release branch
 - If tests have stabilized (they should have by now), you may stop sending the weekly CI report
 
+### Release Cutting - Go or No-Go
+
+Over the release cycle, the release engineering team will cut several releases (alpha, beta, rc), before it ends with a new major Kubernetes release. During the release cycle, tests fail and potentially prevent the cut of a new version. 
+
+Therefore, the key objective is to create a picture if the release branch manager can starting the release process or if there is a major failure that needs to be solved first. To avoid a last-minute No-Go, the assigned CI Signal member aligns at least 2-3 days before the day of the release cut with the [corresponding release branch manager]([https://github.com/kubernetes/sig-release/tree/master/releases](https://github.com/kubernetes/sig-release/tree/master/releases)). Potentially blocking tests have to be followed up with high priority and being driven to be solved until release day. Having a short daily alignment telling the Go/No-Go indication should give an idea if this is going in the right direction. If the CI Signal member decides on a No-Go, the issues have to be addressed. A short heads up about what is blocking, what are the current actions to resolve this and, if possible, an estimation of how long it will take to be solved should be given to the release lead, branch manager and to the test corresponding SIG leads.
```

Comment:
An important part here is that the Branch Manager team also needs to be briefed about the efforts to resolve the blocking tests. In case of a delay, we need to know as many details as possible about the failure and how long to expect for a fix to land. Usually we are involved, but most of the time, the CI Signal has much more context as they track the issues over time. Also, we need to inform the community and I think the information about the failing test should come from the tracking CI Signal team.

Reply:
Indeed, I furthermore would drive it this way to say ether we are able to deliver all insights or in case we don't have the 100% view we involve in the alignment the corresponding person working on the fix.